### DISCUSSION TITLE
Proble with new version iOS

### DISCUSSION THREAD

**Discussion by ari1101** [SafeExamBrowser/seb-mac#131] [Q&A] [answered]
Good evening, recently on one of our tablets with iOS 15 operating systems we received the following message when running the 3.0 secure exam browser, which leaves us wondering the reason for it.

We are looking forward to your comments.

Thanks
![Problem with safe exam browser](https://user-images.githubusercontent.com/91102971/134097480-5396139a-50e7-4d1f-95a6-3cb8c49277fc.jpeg)

**Comment by danschlet** [comment, marked as answer]
In SEB 3.0, a setting needs to be changed in the configuration used for an exam to run on iOS 15. In SEB's in-app-settings, you need to change the option at _Settings / Security / Allow Running on iOS Beta_ to _iOS 15_. In programmatically generated settings (or changing unencrypted config files in a text editor), change the key `allowiOSBetaVersionNumber = 15`.

The update SEB 3.0.1 runs automatically on iOS 15, unfortunately the approval to the App Store is still pending, we hope it will be available later today. On some single machines, you could also install the fully final [TestFlight beta for 3.0.1 with this link](https://testflight.apple.com/join/egxLmc4j).

**Reply by danschlet** [reply]
SEB 3.0.1 was approved to the App Store the same day a week ago, that version runs by default on iOS 15.

**Reply by ari1101** [reply]
thank you very much!
